src/txn/txn_rollback_to_stable.c

```diff
@@ -1623,6 +1623,20 @@ __rollback_progress_msg(WT_SESSION_IMPL *session, struct timespec rollback_start
     }
 }
 
+/*
+ * __check_btree_modified --
+ *     Check that the rollback to stable btree is dirty or not.
+ */
+static int
+__check_btree_modified(WT_SESSION_IMPL *session, const char *uri, bool *modified)
+{
+    WT_DECL_RET;
+
+    ret = __wt_conn_dhandle_find(session, uri, NULL);
+    *modified = ret == 0 && S2BT(session)->modified;
```

Comment:
I re-wrote this logic to use WT_DECL_RET instead of WT_RET so that we always set the modified variable. I'm not sure if this makes the code readability better or worse, @agorrod @kommiharibabu what are your thoughts?

Reply:
With the above change, we don't need to initialize the `modified` flag to false in the caller. I am good with the above change.